internal/envconfig/envconfig.go

```diff
@@ -55,6 +55,8 @@ var (
 	// setting the environment variable "GRPC_EXPERIMENTAL_ENABLE_NEW_PICK_FIRST"
 	// to "true".
 	NewPickFirstEnabled = boolFromEnv("GRPC_EXPERIMENTAL_ENABLE_NEW_PICK_FIRST", false)
+	//HTTPProxy is set to empty string if HTTPS_PROXY env it is not set.
+	HTTPProxy = stringFromEnv("HTTPS_PROXY", "")
```

Comment:
i think we need more details about what this environment variable is used for. May be something like "HTTPSProxy is set to specify the proxy address if proxy should be used."?

Reply:
Done